airflow/utils/logging.py

```diff
@@ -71,7 +71,7 @@ def read(self, remote_log_location, return_error=False):
             try:
                 s3_key = self.hook.get_key(remote_log_location)
                 if s3_key:
-                    return s3_key.get_contents_as_string().decode()
+                    return s3_key.get_contents_as_string()
```

Comment:
At this moment this decode()  breaks  log retrieval  with non-ASCII characters  in python 2.7.